### ISSUE TITLE
Unable to update docker install

### ISSUE THREAD

**Issue by amorphic** [microweber/microweber#897]
I installed Microweber `1.2.11` as a Docker container using the [official image](https://hub.docker.com/r/microweber/microweber/) and it worked fine.

When attempting to follow the [Update guide](https://microweber.org/update-guide) to update my installation to `1.2.19` I encounter an error:

>The standalone update can't be run on this server because:

> The git repository is recognized on your server.

I was able to workaround this by commenting out lines [21-24 in the `standalone-updater` module](https://github.com/microweber-modules/standalone-updater/blob/4a09141f5b5a4f554c3b88134df6b3950179fb90/admin.php#L21):

```
if (is_dir($projectMainDir . DS . '.git')) {
    $canIUpdate = false;
    $canIUpdateMessages[] = 'The git repository is recognized on your server.';
}
```

After doing this ---^ I was able to update to `1.2.19` as expected.

When creating a Docker image, the microweber `Dockerfile` [explicitly copies the entire `microweber` source directory into `/var/www/html`](https://github.com/microweber/microweber/blob/80035efc0d97412d1a8ae5b8a9c22f0efb4052c1/Dockerfile#L30):

```
COPY . /var/www/html
```

So I think the solution might be one of these:

1. The `.git` directory needs to be deleted from the source before the `COPY`
2. The `.git` directory needs to be deleted from the destination after the `COPY`
3. The `.git` check need to be removed from the `standalone-updater` module

**Comment by peter-mw** [comment]
Yes you are correct. the `.git` folder should not be included

I have added a fix in the dev branch here https://github.com/microweber/microweber/blob/dev/Dockerfile#L31

**Comment by peter-mw** [comment]
Fixed in version 1.2.20
